src/main/java/org/usfirst/frc4904/robot/commands/GroundIntake.java

```diff
@@ -0,0 +1,17 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Manipulator;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.standard.commands.solenoid.SolenoidExtend;
+import org.usfirst.frc4904.standard.commands.solenoid.SolenoidRetract;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class GroundIntake extends CommandGroup {
+	public GroundIntake() {
+		addSequential(new SolenoidExtend("ManipulatorGroundOut", RobotMap.Component.manipulator.ground));
+		addSequential(new MotorConstant("ManipulatorRollersIn", RobotMap.Component.manipulator.rollers, Manipulator.INTAKE_SPEED));
```

Comment:
this command never finishes, so the rest of the commands won't finish. This should likely be a runFor

Reply:
also, there should be a command called ManipulatorRollerSet that sets the manipulator roller to a speed, then ManipulatorRollerStop and ManipulatorRollerIntake commands that extend it and set it to INTAKE_SPEED, and STOP SPEED